Edit cart route for testing

alexjcwong opened the pull request:
Logic fully fleshed out, just needs to be tested.

alakhani5 reviewed (commented):
update id in put route to userId

the update for a value in the cart changes between the products in the cart - may need to update the "where" in the findOne to include a productId and an orderId?

alexjcwong commented:
changed the image size so it displayed correctly, updated an image link in the seed so it wasn't a duplicate, and updated the put route. Added a comment to the orderId part of the query. Let me know if that makes sense!

alexjcwong commented:
Updated based on feedback to make sure proper associations were set
